tests/test_upstream_suite.py

```diff
@@ -6,307 +6,273 @@
 from sanity_html import render
 
 
-def test_007_link_mark_def():
-    fixture_data = json.loads((Path(__file__).parent / 'fixtures' / 'upstream' / '007-link-mark-def.json').read_text())
+def get_text(rel_path) -> dict:
+    """Load and return fixture data as dict."""
+    return json.loads((Path(__file__).parent / rel_path).read_text())
+
+
+def test_001_empty_block():
+    fixture_data = get_text('fixtures/upstream/001-empty-block.json')
     input_blocks = [fixture_data['input']] if type(fixture_data['input']) == dict else fixture_data['input']
     expected_output = fixture_data['output']
     output = render(input_blocks)
-    assert expected_output == output
+    assert output == expected_output
```

Comment:
The original assert statement made the pycharm side comparison very confusing

![image](https://user-images.githubusercontent.com/25310870/114751258-357b8300-9d55-11eb-915c-703429b850ce.png)

Now they match